plugin/yaml/tests/sample_plugin_without_version.yaml

```diff
@@ -3,7 +3,6 @@ description: Testing
 plugintype: task
 pluginmods:
   - cli
-pluginversion: ""
```

Comment:
why ?

Reply:
renamed to `sample_plugin_empty_version`